app/dawn-hub/Views/OnboardingViews/Step2_BlockSettingsView/BlockSettingsView.jsx

```diff
@@ -87,21 +104,25 @@ class BlockSettingsView extends Component {
 
 			let blockingPolicy;
 			switch (kindsOfTrackers) {
-				case 0:
+				case 1:
 					blockingPolicy = 'BLOCKING_POLICY_EVERYTHING';
 					break;
-				case 1:
+				case 2:
 					blockingPolicy = 'BLOCKING_POLICY_RECOMMENDED';
 					break;
-				case 2:
+				case 3:
 					blockingPolicy = 'BLOCKING_POLICY_NOTHING';
 					break;
 				default:
 					break;
 			}
 			setBlockingPolicy({ blockingPolicy });
 
```

Comment:
The argument values passed to the `renderAnswerBlock` helper for the blocking policy radio boxes need to be updated to match

Reply:
Yes they do. Updated :D